app/src/main/res/layout/layout_nav_header.xml

```diff
@@ -24,7 +24,7 @@
         android:layout_height="wrap_content"
         android:layout_width="match_parent"
         android:paddingTop="@dimen/nav_header_vertical_spacing"
-        android:text="Rajan Maurya"
+        android:text="@string/rajan_maurya"
```

Comment:
Here should not be my name and email, we will show logged in user name and email. Please make string with name `username = Username` and `email = Email` and change it.

Reply:
updated